### PR TITLE
Auxtel ronchi

### PR DESCRIPTION
**Pull request by jeremyneveu**
Tune spectractor parameters for Ronchi gratings
and some improvements (mask, plots)

### CODE REVIEW

File: spectractor/extractor/extractor.py [hunk 1]
```diff
@@ -238,12 +239,14 @@ def set_y_c(self):
                 else:
                     self.p[k] = 0
 
-    def set_mask(self, params=None):
+    def set_mask(self, params=None, fwhmx_clip=3*parameters.PSF_FWHM_CLIP, fwhmy_clip=parameters.PSF_FWHM_CLIP):
```
Comment (mfisherlevine) [inline spectractor/extractor/extractor.py:242]: Having complicated defaults in function definitions is often quite dangerous, but I think we should save that for a later cleanup, as that's done in quite a lot of places already.

File: spectractor/extractor/extractor.py [hunk 2]
```diff
@@ -693,12 +707,17 @@ def adjust_spectrogram_position_parameters(self):
...
+        #     for k, par in enumerate(self.input_labels):
+        #         if "y_c" in par and "_2" not in par:
+        #             self.fixed[k] = False
         self.sparse_indices = None
-        run_minimisation(self, "newton", epsilon, self.fixed, xtol=1e-4, ftol=100 / self.data.size)
+        run_minimisation(self, "newton", epsilon, self.fixed, xtol=1e-2, ftol=0.01)  # 1000 / self.data.size)
         self.fixed = fixed_default
         self.sparse_indices = None
-        self.set_mask(params=self.p)
+        self.set_mask(params=self.p, fwhmx_clip=3*parameters.PSF_FWHM_CLIP, fwhmy_clip=parameters.PSF_FWHM_CLIP)
```
Comment (mfisherlevine) [inline spectractor/extractor/extractor.py:720]: If you are going to have expressions in default args then reasserting them downstream isn't usually a good pattern. Happy to have a chat about all of this when we next speak though, so it's probably fine for now.

File: spectractor/extractor/images.py [hunk 3]
```diff
@@ -729,7 +729,9 @@ def load_AUXTEL_image(image):  # pragma: no cover
         image.target_guess = [parameters.CCD_IMSIZE - float(image.header["OBJECTY"]),
                               parameters.CCD_IMSIZE - float(image.header["OBJECTX"])]
     image.disperser_label = image.header["GRATING"]
-    parameters.DISTANCE2CCD = 113 + float(image.header["LINSPOS"])  # mm
+    parameters.DISTANCE2CCD = 115 + float(image.header["LINSPOS"])  # mm
```
Comment (mfisherlevine) [inline spectractor/extractor/images.py:732]: Fine, I just need to make a similar adjustment on my side, so just noting this to myself.

**Review by mfisherlevine — APPROVED** [review]

**Comment by mfisherlevine** [comment]
Obviously the comments/approval above are modulo the fact the tests are currently not passing.

**Comment by jeremyneveu** [comment]
Yes indeed, it is just the coverage that has decreased by 0.001%... I think the most important for you in the LINSPOS change has you noticed in your comments.

**Pull request merged**